dali/plugin/plugin_manager.cc

```diff
@@ -18,9 +18,11 @@
 
 namespace dali {
 
-void PluginManager::LoadLibrary(const std::string& lib_path) {
+void PluginManager::LoadLibrary(const std::string& lib_path, bool global_symbols) {
```

Comment:
Maybe we can pass `flags` directly to that funciton?

Reply:
We could. We'd still need a Python interface, though.

Reply:
Ok, so let us leave it as it is.